src/features/network-modifications/twoWindingsTransformer/common/TwoWindingsTransformerDialogHeader.tsx

```
@@ -0,0 +1,77 @@
+
+export function TwoWindingsTransformerDialogHeader({
+    twtToModify,
+    isModification = false,
+}: Readonly<TwoWindingsTransformerDialogHeaderProps>) {
+    const equipmentId = useWatch({ name: FieldConstants.EQUIPMENT_ID });
+
+    const twtIdField = isModification ? (
+        <TextField
+            size="small"
+            fullWidth
+            label={'ID'}
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)